.htaccess

```diff
@@ -23,3 +23,5 @@ Redirect 301 /dll.html http://wiki.dlang.org/Win32_DLLs_in_D
 Redirect 301 /htomodule.html http://wiki.dlang.org/Converting_C_.h_Files_to_D_Modules
 Redirect 301 /memory.html http://wiki.dlang.org/Memory_Management
 Redirect 301 /windows.html http://wiki.dlang.org/D_for_Win32
+
+RewriteRule ^changelog.html$ changelog/legacy.html [R=301,L]
```

Comment:
We can use either. But we'll need to test if the `^` will work as expected, IIRC there were some issues with `RewriteBase`.